### ISSUE TITLE
why my keras only show val_acc, val_loss at the end of epoch

### ISSUE THREAD

**Issue by wangxupeng** [keras-team/keras#10790]
29696/57000 [==============>...............] - ETA: 1s - loss: 0.0372 - acc: 0.9886
30464/57000 [===============>..............] - ETA: 1s - loss: 0.0374 - acc: 0.9884
31232/57000 [===============>..............] - ETA: 1s - loss: 0.0382 - acc: 0.9883
32000/57000 [===============>..............] - ETA: 1s - loss: 0.0380 - acc: 0.9883
32768/57000 [================>.............] - ETA: 1s - loss: 0.0379 - acc: 0.9884
33536/57000 [================>.............] - ETA: 1s - loss: 0.0384 - acc: 0.9882
34304/57000 [=================>............] - ETA: 1s - loss: 0.0380 - acc: 0.9883
35072/57000 [=================>............] - ETA: 1s - loss: 0.0377 - acc: 0.9884
35840/57000 [=================>............] - ETA: 1s - loss: 0.0377 - acc: 0.9884
36608/57000 [==================>...........] - ETA: 1s - loss: 0.0378 - acc: 0.9883
37376/57000 [==================>...........] - ETA: 1s - loss: 0.0379 - acc: 0.9883
38272/57000 [===================>..........] - ETA: 1s - loss: 0.0384 - acc: 0.9882
39040/57000 [===================>..........] - ETA: 1s - loss: 0.0381 - acc: 0.9883
39808/57000 [===================>..........] - ETA: 1s - loss: 0.0379 - acc: 0.9884
40576/57000 [====================>.........] - ETA: 1s - loss: 0.0377 - acc: 0.9885
41472/57000 [====================>.........] - ETA: 1s - loss: 0.0377 - acc: 0.9885
42240/57000 [=====================>........] - ETA: 0s - loss: 0.0376 - acc: 0.9885
43008/57000 [=====================>........] - ETA: 0s - loss: 0.0377 - acc: 0.9884
43776/57000 [======================>.......] - ETA: 0s - loss: 0.0377 - acc: 0.9885
44544/57000 [======================>.......] - ETA: 0s - loss: 0.0379 - acc: 0.9884
45312/57000 [======================>.......] - ETA: 0s - loss: 0.0379 - acc: 0.9885
46080/57000 [=======================>......] - ETA: 0s - loss: 0.0378 - acc: 0.9885
46848/57000 [=======================>......] - ETA: 0s - loss: 0.0379 - acc: 0.9884
47616/57000 [========================>.....] - ETA: 0s - loss: 0.0382 - acc: 0.9883
48384/57000 [========================>.....] - ETA: 0s - loss: 0.0382 - acc: 0.9883
49152/57000 [========================>.....] - ETA: 0s - loss: 0.0380 - acc: 0.9884
50048/57000 [=========================>....] - ETA: 0s - loss: 0.0379 - acc: 0.9884
50816/57000 [=========================>....] - ETA: 0s - loss: 0.0380 - acc: 0.9883
51584/57000 [==========================>...] - ETA: 0s - loss: 0.0382 - acc: 0.9883
52352/57000 [==========================>...] - ETA: 0s - loss: 0.0382 - acc: 0.9883
53120/57000 [==========================>...] - ETA: 0s - loss: 0.0382 - acc: 0.9883
54016/57000 [===========================>..] - ETA: 0s - loss: 0.0381 - acc: 0.9883
54784/57000 [===========================>..] - ETA: 0s - loss: 0.0380 - acc: 0.9884
55552/57000 [============================>.] - ETA: 0s - loss: 0.0380 - acc: 0.9883
56320/57000 [============================>.] - ETA: 0s - loss: 0.0382 - acc: 0.9883
57000/57000 [==============================] - 4s 69us/step - loss: 0.0383 - acc: 0.9882 - val_loss: 0.0636 - val_acc: 0.9837

my model complie and fit part like this:

network.compile(optimizer='rmsprop',
                loss='categorical_crossentropy',
                metrics=['accuracy'])
network.fit(train_images, train_labels, epochs=5, batch_size=128,validation_split=0.05)

why val_loss and val_acc don't show like acc

**Comment by chriskoups** [comment]
Here is not a place for help and this is not a bug or feature.
Please do not post here this type of questions in github. Use stackoverflow.
The validation is executed only when the training finishes in order to VALIDATE the training.
Please close the issue by pressing the close button under the reply/comment box.
